### ISSUE TITLE
One small enhancement for much better workflow with --serialize

### ISSUE THREAD

**Issue by JustACuteGirlLookingForAManToFixHerBugs** [beat-machine/beat-machine#55]
Working with serialized .beat files is like 20 times faster on my machine. It's a no-brainer. 
But! Right now one needs to juggle between .mp3 and .beat inputs. And if you forget that you already have a pre-made .beat file for that particular song, you end up re-creating it. 

If it's possible to **check whether in.beat file already exists**, then it solves both problems at once.

So, the logic:
1. User inputs command: `py beatmachine --input in.mp3 --effects effects.json --output out.mp3 --serialize`
2. Program checks if in.beat already exists. If it does - it skips serializing, and uses in.beat for the input. Prints something like "Found an existing in.beat file. Using it instead of in.mp3"
3. If it doesn't - continues as usual, creating in.beat from in.mp3. 
4. Outputs out.mp3

If user doesn't use --serialize command - nothing changes, it just uses in.mp3 and won't check for in.beat.

**Comment by branchpanic** [comment]
I agree that saving the serialized beat file is a sensible default. It does the same computation anyway.
